files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -255,22 +255,28 @@
   relabel_configs:
   - source_labels: [__address__]
 [[- if .CAPICluster ]]
-    replacement: $1:10257
+    replacement: 10257
```

Comment:
LGTM but can you keep the instance label anyway ?
Also, should we not do the same for the scheduler ?